app/views/pages/robots.text.erb

```diff
@@ -2,3 +2,5 @@
 
 User-agent: *
 Disallow: /
```

Comment:
Do we need to allow anything for search.gov here?

Reply:
Yes, update incoming